stable/reloader/templates/rolebinding.yaml

```diff
@@ -17,7 +17,7 @@ metadata:
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
-  name: {{ template "reloader-name" . }}-role
+  name: {{ template "reloader-fullname" . }}-role
```

Comment:
@winromulus as you are using `reloader-fullname` template in all files then kindly remove the `reloader-name` template from `reloader/templates/_helpers.tpl` file as it not being used anywhere.